index.js

```diff
@@ -8,8 +8,9 @@ const db = require('./db/models.js');
 app.use(express.static(path.join(__dirname, './client/public')));
 app.use(express.urlencoded({ extended: false }));
 
-app.get('/author', (req, res) => {
-  db.getAuthorInfo(14, (err, results) => {
+app.post('/author', (req, res) => {
```

Comment:
Why did you use a POST request if you're fetching data?

Reply:
My thinking is that I'll need to grab the unique/book ID from the URL and send it to the endpoint to query the db, so I am using POST to send that data (ID).

Reply:
Gotcha. I'm not 100% sure, but I think a GET request with a query string in the request URL would make more sense because get requests indicates that you're fetching data, but POST requests as far as I understand implies a change on the database of some sort. Anyway, other than that, you're good to merge.

Reply:
Yes, your rationale for POST makes sense. For expediency, I'll just leave it as a POST request and insert a comment for myself that I may need (or should) to change it a GET request later on.